README.md

```diff
@@ -3,4 +3,34 @@ Juntos is a web application that lets you co watch streaming service such as you
 
 # Artefacts  
 * [Product Requirement Document](/prd/Juntos_PRD.pdf) 
-* [System Design](/system-design/Juntos_system_design.pdf)
\ No newline at end of file
+* [System Design](/system-design/Juntos_system_design.pdf)
+
+
+# Youtube API - seekTo()
+
+## Overview
+`seekTo()` function is available on the player object and is used to play the video from the timestamp specified - `seekTo(timestamp)`.
+
+## Use Cases  
+seekTo() function can be used in the following scenarios  
+* When a user joins late 
+
+    When the video starts playing the video sync service on the client side will keep track of current play progress. The progress will be shared to the backend by all clients and the backend video sync service will check if all users are on the same progress. When a user joins late, on joining, the client video sync service will get the current video progress(timestamp) from the backend video sync service. On recieveing the timestamp, `seekTo()` will be executed post which `playVideo()` will be executed.This will sync the late joining user to the current video progress of the group.  
+
+* When user changes the progress of the video  
+
+    When the host of the group decides to play video from a non-zero timestamp, i.e the user changes progress bar position. An event* is fired that communicates the current progress to the backend which further communicates this with all the other connected clients. Other connected clients on recieving this information will execute `seekTo()` followed by `playVideo()`.
+    
+### Flow Diagram
+![alt text](flow-diagrams/seekTo-flow.png)
```

Comment:
Please add a link of the diagram, example: lucid chart link

Reply:
sure